fe/fe-core/src/main/java/org/apache/doris/catalog/MTMV.java

```diff
@@ -196,9 +196,10 @@ public void addTaskResult(MTMVTask task, MTMVRelation relation,
                 // The replay thread may not have initialized the catalog yet to avoid getting stuck due
                 // to connection issues such as S3, so it is directly set to null
                 if (!isReplay) {
+                    ConnectContext currentContext = ConnectContext.get();
```

Comment:
ConnectContext.get() is created by org.apache.doris.mtmv.MTMVPlanUtil#createMTMVContext,are you sure use it?